bug: fl crashes with no arguments

```
> fl 
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
fl/helpers.ConfParse({0x14000012090?, 0x0?, 0x14000065f38?}, 0x100bb0994?)
	/Users/runner/work/fl/fl/src/helpers/argparse.go:141 +0x194
main.main()
	/Users/runner/work/fl/fl/src/fl.go:123 +0xac
```

should add a test for this too

--- a/src/helpers/argparse.go
+++ b/src/helpers/argparse.go
@@ -138,7 +138,7 @@ var (
 
 // check if this is a config command - follow format 'fl config <CONFIGCMD>'
 func ConfParse(args []string, Config *io.Config) (confCmd bool, err error) {
-	if args[1] == "conf" {
+	if len(args) > 1 && args[1] == "conf" {
 		if regex_autoexecute.MatchString(args[2]) {
 			confHandlerAutoexec(Config, args[2])
 		} else if regex_language.MatchString(args[2]) {

